.github/workflows/release.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       status_url: 'https://api.telescope.cdot.systems/v1/status'
       supabase_url: 'https://api.telescope.cdot.systems/v1/supabase'
       anon_key: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UiLAogICAgImlhdCI6IDE2NDg2MTI4MDAsCiAgICAiZXhwIjogMTgwNjM3OTIwMAp9.YMOYR_8pZqqDRJkoAFuEWp-FaQfvmHXTOXKncBlRjdE'
+      dependency_discovery_url: 'https://api.telescope.cdot.systems/v1/dependency-discovery'
```

Comment:
Move this up with the other URLs (below `status_url`) vs. with the supabase stuff.